ci_framework/hooks/templates/service_config_ceph.yaml.j2

```diff
@@ -0,0 +1,71 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- ./{{ cifmw_kustomize_cr_file_name }}
+namespace: {{ cifmw_install_yamls_defaults['NAMESPACE'] }}
+patches:
+- patch: |-
+    - op: add
+      path: /spec/extraMounts
+      value:
+        - name: v1
+          region: r1
+          extraVol:
+          - propagation:
+            - Glance
+            - volume1
+            - CinderBackup
+            extraVolType: Ceph
+            volumes:
+            - name: ceph
+              projected:
+                sources:
+                - secret:
+                    name: ceph-conf-files
+            mounts:
+            - name: ceph
+              mountPath: "/etc/ceph"
+              readOnly: true
```

Comment:
It is already defined here https://github.com/openstack-k8s-operators/openstack-operator/blob/main/config/samples/core_v1beta1_openstackcontrolplane_network_isolation_ceph.yaml#L8 Do we still need to customize it?

Reply:
Sounds good to have a new scenario with galera and ceph. We can add a new cr in the openstack-operator with galera ceph scenario.